QuantConnect.TradeStationBrokerage/TradeStationBrokerage.cs

```diff
@@ -327,9 +340,8 @@ public override bool PlaceOrder(Order order)
         foreach (var brokerageOrder in response.Orders)
         {
             order.BrokerId.Add(brokerageOrder.OrderID);
-
             // Check if the order failed due to an existing position. Reason: EC701: You are long/short N shares.
-            if (brokerageOrder.Message.Contains("EC701", StringComparison.InvariantCultureIgnoreCase))
+            if (PlaceOrderTradeStationErrorCodes.Any(item => brokerageOrder.Message.Contains(item, StringComparison.InvariantCultureIgnoreCase)))
```

Comment:
Review if we can revert the logic, if message is empty or success assume all is good, else it's a failure

Reply:
![image](https://github.com/QuantConnect/Lean.Brokerages.TradeStation/assets/45608740/94fff19f-e2de-48ef-b4fb-be04a8535c4e)

Reply:
We can validate on string contain `Order failed.`